suite/spdz/src/test/java/dk/alexandra/fresco/suite/spdz/storage/DummyDataSupplierImpl.java

```diff
@@ -59,41 +59,36 @@ public DummyDataSupplierImpl(int myId, int numberOfPlayers) {
   public SpdzTriple getNextTriple() {
     if (numberOfPlayers == 2) {
       return triples_2party.get(myId);
-    } else if (numberOfPlayers == 3) {
+    } else {
       return triples_3party.get(myId);
     }
-    throw new RuntimeException("Not implemented yet");
```

Comment:
I think it's nice to have an exception cast if someone tries to use the dummy supplier for more than 3 parties. As it stands, I guess an array out of bounds exception is cast if player >3 tries to call any method within this class.

Reply:
The RuntimeExceptions could never be reached. The constructor already does a check for numberOfParties > 3 (and casts a IllegalArgumentException).

Reply:
Ah, nice. Well then, if tests go through, all is good by me.